packages/rsu_client.rb

```diff
@@ -0,0 +1,73 @@
+      system "sed -i 's,#{CREW_PREFIX},/usr,gw /dev/stdout' ./rsu/framework/modules/updater/download/wxsysdload.pm"
+      system "sed -i 's,#{CREW_PREFIX},/usr,gw /dev/stdout' ./rsu/framework/modules/updater/gui/zenity.pm"
+      system "sed -i 's,#{CREW_PREFIX},/usr,gw /dev/stdout' ./rsu/rsu-query"
+      system "sed -i 's,#{CREW_PREFIX},/usr,gw /dev/stdout' ./runescape"
+      system "sed -i 's,#{CREW_PREFIX},/usr,gw /dev/stdout' ./templates/packaging/usr/games/rsu-settings"
+      system "sed -i 's,#{CREW_PREFIX},/usr,gw /dev/stdout' ./templates/packaging/usr/games/runescape"
+      system "sed -i 's,#{CREW_PREFIX},/usr,gw /dev/stdout' ./templates/packaging/usr/games/update-runescape-client"
+      system "sed -i 's,#{CREW_PREFIX},/usr,gw /dev/stdout' ./templates/packaging/usr/local/bin/xdg-open"
+      system "sed -i 's,#{CREW_PREFIX},/usr,gw /dev/stdout' ./templates/protocol/install-jagex-protocol-linux"
+      system "sed -i 's,#{CREW_PREFIX},/usr,gw /dev/stdout' ./updater"
+
+      system "sed -i 's,/usr,#{CREW_PREFIX},gw /dev/stdout' ./changelog.txt"
+      system "sed -i 's,/usr,#{CREW_PREFIX},gw /dev/stdout' ./install-desktop-icons"
+      system "sed -i 's,/usr,#{CREW_PREFIX},gw /dev/stdout' ./rsu-settings"
+      system "sed -i 's,/usr,#{CREW_PREFIX},gw /dev/stdout' ./rsu/bin/rsu-query-unix"
+      system "sed -i 's,/usr,#{CREW_PREFIX},gw /dev/stdout' ./rsu/framework/API/client/launch/launcher.pm"
+      system "sed -i 's,/usr,#{CREW_PREFIX},gw /dev/stdout' ./rsu/framework/Perl5lib/XML/RSSLite.pm"
+      system "sed -i 's,/usr,#{CREW_PREFIX},gw /dev/stdout' ./rsu/framework/modules/rsu/java/jre.pm"
+      system "sed -i 's,/usr,#{CREW_PREFIX},gw /dev/stdout' ./rsu/framework/modules/rsu/mains.pm"
+      system "sed -i 's,/usr,#{CREW_PREFIX},gw /dev/stdout' ./rsu/framework/modules/rsu/nvidia/optimus.pm"
+      system "sed -i 's,/usr,#{CREW_PREFIX},gw /dev/stdout' ./rsu/framework/modules/updater/download/sysdload.pm"
+      system "sed -i 's,/usr,#{CREW_PREFIX},gw /dev/stdout' ./rsu/framework/modules/updater/download/wxsysdload.pm"
+      system "sed -i 's,/usr,#{CREW_PREFIX},gw /dev/stdout' ./rsu/framework/modules/updater/gui/zenity.pm"
+      system "sed -i 's,/usr,#{CREW_PREFIX},gw /dev/stdout' ./rsu/rsu-query"
+      system "sed -i 's,/usr,#{CREW_PREFIX},gw /dev/stdout' ./runescape"
+      system "sed -i 's,/usr,#{CREW_PREFIX},gw /dev/stdout' ./templates/packaging/usr/games/rsu-settings"
+      system "sed -i 's,/usr,#{CREW_PREFIX},gw /dev/stdout' ./templates/packaging/usr/games/runescape"
+      system "sed -i 's,/usr,#{CREW_PREFIX},gw /dev/stdout' ./templates/packaging/usr/games/update-runescape-client"
+      system "sed -i 's,/usr,#{CREW_PREFIX},gw /dev/stdout' ./templates/packaging/usr/local/bin/xdg-open"
+      system "sed -i 's,/usr,#{CREW_PREFIX},gw /dev/stdout' ./templates/protocol/install-jagex-protocol-linux"
+      system "sed -i 's,/usr,#{CREW_PREFIX},gw /dev/stdout' ./updater"
```

Comment:
Better solution?
```
      system "for file in \$(find . -type f); do
                sed -i 's,/usr,#{CREW_PREFIX},g' \${file}
              done"
```

Reply:
That would include binary files. The list I got came from searching for `/usr/bin`, for some reason searching for `/usr` causes issues. I'll try to come up with proper patches in the future, but for now it works.

Reply:
Alright, I will.
Here's a translation of my search script, if needed.
```ruby
system "for f in \$(grep -r /usr/bin .|egrep -v '^Binary'|egrep -o '^[^:]+');do
          sed -i 's,/usr,#{CREW_PREFIX},gw /dev/stdout' $f
        done"
```

Reply:
Nice suggestion.

Reply:
@JL2210 your version gives these grep warnings:
```
grep: ./share/img/Custom: No such file or directory
grep: Icons.txt: No such file or directory
```
Looks like a quoting issue, but I couldn't figure out how to fix it.

Reply:
My solution, on the other hand, surprisingly works!

Reply:
All right. You don't need to brag. ;P